### PR TITLE
Add press release + press kit on launch of Haiku merchandise

### PR DESCRIPTION
**Pull request by jt15s**
Added a press release and press kit on the launch of the official Haiku merchandise in partnership with Freewear as well as a prominent link to buy merchandise on the website homepage. Please feel free to make any design changes to the homepage link if you think it needs further improvement.

### CODE REVIEW

File: content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+FreeWear has greatly simplified the process of selling merchandise, having built their own online store, and handling the printing and shipping processes as well as the process of donating a portion of the sales straight to the FOSS project/organisation. Moreover, most of the world can receive merchandise from FreeWear. A partnership with FreeWear does not involve any upfront costs from the FOSS project, as a portion of the sales pays from expenses incurred such as printing.   
+
+Any FOSS organisations who are interested in working with FreeWear can contact them through email at admin(@)freewear.org   
+
+A list of the available merchandise, as well as the prices in Euros (EUR) and US Dollars (USD), follows, along with an image showing all the available products. Note that prices in US Dollars vary depending on exchange rates and are indicative only.
```
Comment (pulkomandy) [inline content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md:19]: Do we really need this here? I would just put a link to the Freewear store so people can check it out for themselves.

Maybe it's different for the press release to send to other places, but on the website we don't need all the info here.

File: content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+FreeWear has greatly simplified the process of selling merchandise, having built their own online store, and handling the printing and shipping processes as well as the process of donating a portion of the sales straight to the FOSS project/organisation. Moreover, most of the world can receive merchandise from FreeWear. A partnership with FreeWear does not involve any upfront costs from the FOSS project, as a portion of the sales pays from expenses incurred such as printing.
```
Comment (pulkomandy) [inline content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md:15]: All of this is not really different from the previous store at cafepress.

The main changes for this new offer are:
- Transparency on the prices: it is clear how much money is donated to Haiku for each item
- Freewear also has partnerships with some events like Capitole du Libre, meaning people will be able to buy our merch in person in such events (sponsoring both the event and Haiku, and saving on shipping costs)

File: content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+[**Anyone who is interested in purchasing the merchandise may place orders at Freewear's online store here**](https://www.freewear.org/Haiku).
+
+**ENDS**
+
+For media enquiries, please contact:   
+contact(@)haiku-os.org (The Haiku Project)   
+admin(@)freewear.org (FreeWear)  
+
+
+## NOTES TO EDITORS
```
Comment (pulkomandy) [inline content/news/2021-12-01_media_release_the_haiku_project_and_freewear_launch_official_haiku_merchandise.md:54]: This and all things below are fine for the press release, but should probably not be copied on our own website?

**Review by pulkomandy — COMMENTED** [review]
Looks fine as a press release, but do we want to publish the press release as-is on the website? I would publish a more website-ish thing on the website and send the press release directly to, well, the press? How do these things usually work?

**Comment by kallisti5** [comment]
Does Haiku, Inc. hold a copy of the login credentials for FreeWear?   We'll need access for oversight + tax reporting.

**Comment by jt15s** [comment]
@pulkomandy I think with the Beta3 release we did this too, but you have a good point - what I'll do is I'll remove the parts more relevant to the press release as you suggested, plus add those extra points on Freewear's benefits. As to whether we should call it "new" official merchandise, most people don't seem to know that a Cafepress store exists, plus the fact that we're launching it already implies it's new.

**Comment by jt15s** [comment]
@kallisti5 Yeah, what I've been doing actually for all the Inc stuff is conversing with @leavengood on the Haiku forums, I actually raised this with him and it was determined that the login would need to be sent through some encrypted platform (I know the Inc uses KeyBase). Would sending it to you through Matrix work? I've just sent an invite on there.

**Pull request merged**